src/common/catalog/catalog-entity.ts

```diff
@@ -62,6 +68,19 @@ export interface CatalogCategorySpec {
      */
     kind: string;
   };
+
+  /**
+   * These are the columns used for displaying entities when in the catalog.
+   *
+   * If this is not provided then some default columns will be used, similar in
+   * scope to the columns in the "Browse" view.
+   *
+   * Even if you provide columns, a "Name" column will be provided as well with
+   * `priority: 0`.
```

Comment:
I wonder if we should let priority 0 take over the name. Now it is impossible to put column on the left side of name.

Reply:
No there is a way, negative numbers.

Reply:
oh alright, got it 👍